cmd/deployment-check/run_check.go

```diff
@@ -0,0 +1,290 @@
+// Copyright 2018 Comcast Cable Communications Management, LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// runDeploymentCheck sets up a deployment and applies it to the cluster.
+// Sets up a deployment and service and uses the client to deploy the test deployment and service.
+// Attempts to hit the service hostname endpoint, looking for a 200 and reports a success if able to retrieve a 200.
+func runDeploymentCheck() error {
```

Comment:
there isn't a case where returning an error is valuable in this code, so remove the `error` return from the signature

Reply:
removed